src/test/scala/ylabs/messaging/ClientTest.scala

```diff
@@ -149,6 +152,130 @@ class ClientTest extends WordSpec with Matchers with BeforeAndAfterEach {
     }
   }
 
+  //Domain added to user name
+  "with user domain connects to the xmpp server"  in new FixtureWithDomain {
```

Comment:
try to not use comments too much - the code should be self explanatory, and comments get often ignored during refactorings